sql/06_create_refresh_tokens.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE refresh_tokens (
+    user_id INTEGER PRIMARY KEY,
```

Comment:
Primary keys need to be unique. we should have an id that is auto generated and allow multiple user_ids with different tokens.

Reply:
Done.